frameworks/angular/workspace/public/view2.fin.json

```diff
@@ -0,0 +1,7 @@
+{
+  "url": "http://localhost:4200/view2",
+  "fdc3InteropApi": "1.2",
```

Comment:
As we are doing an update can we update it to 2.0

Reply:
Done